src/OrbitGl/App.cpp

```diff
@@ -599,11 +599,7 @@ void OrbitApp::OnErrorEnablingUserSpaceInstrumentationEvent(
     main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kSevereWarning,
                                      GetCaptureTimeAt(error_event.timestamp_ns()), message);
     if (!IsLoadingCapture()) {
-      constexpr const char* kDontShowAgainErrorEnablingUserSpaceInstrumentationWarningKey =
-          "DontShowAgainErrorEnablingUserSpaceInstrumentationWarning";
-      main_window_->ShowWarningWithDontShowAgainCheckboxIfNeeded(
-          "Could not enable user space instrumentation", message,
-          kDontShowAgainErrorEnablingUserSpaceInstrumentationWarningKey);
+      main_window_->ShowWarning("Could not enable user space instrumentation", message);
```

Comment:
There's `OrbitApp::SendWarningToUi`. I would just use that instead of duplicating.

Reply:
It uses a callback instead of a method of `MainWindowInterface`, but unfortunately this part of the codebase is messy, so I would avoid duplicating unless we take the time to clean up the existing method.

Reply:
< ... OrbitApp::SendWarningToUi ...

Thanks, I wasn't aware of that.